### PR TITLE
Car hire smoke test implementing in process.

### PR DESCRIPTION
**Pull request by kit-91-daniil**
Fix utils.text_formatter.TextFormatter.format_date_time documentation.

Implement smoke test for car hire

Prepare for rebase

### CODE REVIEW

File: utils/browser_helper.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from loguru import logger
 from selenium.webdriver.common.by import By
 from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.common.by import By
```
Comment (relaxing-observer) [inline utils/browser_helper.py:4]: duplicate

**Review by relaxing-observer — APPROVED** [review]
Good job!

File: config.py [hunk 2]
```diff
@@ -1,11 +1,11 @@
 class BrowserHelperSettings:
-    WAIT_TIME_SECS = 5
+    WAIT_TIME_SECS = 13
```
Comment (relaxing-observer) [inline config.py]: Is that correct? Do we have place where we need to wait 13 sec?

File: tests/car_search_tests/test_car_search.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        main_page.log_out()
+
+    @pytest.mark.smoke
+    @pytest.mark.car_hire
+    @pytest.mark.parametrize("airport, airport_code, pick_up_date, pick_up_time, drop_off_date, drop_off_time",
+                             Data.car_hire_data)
+    def test_user_can_hire_car(self, main_page, main_page_car_hire_tab, trip_viewer_page_car_hire_tab,
+                               trip_viewer_page_overview_tab, trip_viewer_page_header, payment_page,
+                               airport, airport_code, pick_up_date, pick_up_time, drop_off_date, drop_off_time):
+        assertions = Assertions(main_page.browser)
```
Comment (relaxing-observer) [inline tests/car_search_tests/test_car_search.py]: We use assertions as a fixture in coftest. I suppose that you should to remove that line

File: utils/logging.py [hunk 4]
```diff
@@ -18,6 +18,6 @@ def set_logger(self):
                           filter=self.debug_only), \
                logger.add("logs/errors.log",
                           format="{time:DD:MM:YYYY:h:m:s} {level} {message}",
+                          enqueue=True,
```
Comment (yefim-mironyuk) [inline utils/logging.py]: No need in enqueue

**Review by yefim-mironyuk — APPROVED** [review]

File: tests/pages/main_page.py [hunk 5]
```diff
@@ -59,35 +59,36 @@ def choose_depart_date(self, date: str):
...
 
-    def choose_pick_up_location(self, pick_up_location):
+    def choose_pick_up_location(self, pick_up_location, airport_code):
         self.element_interactions.send_text_in_field(SearchCarsTabLocators.CAR_LOCATION, pick_up_location)
-        self.element_interactions.click_element(SearchCarsTabLocators.FIRST_ITEM_LOCATION_DROPDOWN)
-        
+        airport_item_in_ddm_locator = SearchCarsTabLocators.get_airport_item_in_dropdown(airport_code)
+        self.element_interactions.click_element(airport_item_in_ddm_locator)
+
     def choose_date(self, date: str):
```
Comment (relaxing-observer) [inline tests/pages/main_page.py]: It's a duplicate method from class MainPage. As I remember, we decided to move this method to MainPage class from MainPageCarHireTab

File: tests/pages/main_page.py [hunk 6]
```diff
@@ -98,13 +99,13 @@ def choose_drop_off_date(self, date: str):
         self.element_interactions.click_element(SearchCarsTabLocators.DROP_OFF_FORM)
         self.choose_date(date)
```
Comment (relaxing-observer) [inline tests/pages/main_page.py:92]: MainPage.choose_date(self, date)
and etc below in class

File: tests/car_search_tests/test_car_search.py [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+        main_page.open_car_hire_tab()
+        main_page_car_hire_tab.perform_car_sharing(airport, airport_code, pick_up_date, pick_up_time,
+                                                   drop_off_date, drop_off_time)
+        # Move to the trip_viewer_page_car_hire_tab
+        assertions.element_should_contain_value(CarHireTabLocators.HEADER_TITLE, "Car Hire")
```
Comment (relaxing-observer) [inline tests/car_search_tests/test_car_search.py]: Maybe we should throw out "Car hire" to test data as parameter?

File: utils/text_formatter.py [hunk 8]
```diff
@@ -1,14 +1,36 @@
 from collections import namedtuple
-from datetime import date
+from datetime import date, datetime
 
 
 class TextFormatter:
     formatted_date_nt = namedtuple('date_namedtuple', 'year, month_number, month_name, day')
 
     @classmethod
     def date_formatter(cls, date_str: str) -> formatted_date_nt:
```
Comment (relaxing-observer) [inline utils/text_formatter.py]: maybe format_date name?  you said about it yesterday

**Review by relaxing-observer — CHANGES_REQUESTED** [review]

File: config.py (hunk 2 above)
Comment (yefim-mironyuk) [reply config.py]: > Is that correct? Do we have place where we need to wait 13 sec?

Why not. Maybe someone will run this test with slow internet connection)

**Review by yefim-mironyuk — APPROVED** [review]

**Review by relaxing-observer — APPROVED** [review]

**Review by TA-user — APPROVED** [review]

**Pull request merged**